src/thermo/CoverageDependentSurfPhase.cpp

```diff
@@ -0,0 +1,571 @@
+/**
+ *  @file CoverageDependentSurfPhase.cpp
+ *  Definitions for a thermodynamics model of a coverage-dependent surface
+ *  phase derived from SurfPhase, applying adsorbate lateral interaction
+ *  correction factors to the SurfPhase thermodynamic properties.
+ *  (see \ref thermoprops and class
+ *  \link Cantera::CoverageDependentSurfPhase CoverageDependentSurfPhase\endlink).
+ */
+
+// This file is part of Cantera. See License.txt in the top-level directory or
+// at https://cantera.org/license.txt for license and copyright information.
+
+#include "cantera/thermo/CoverageDependentSurfPhase.h"
+#include "cantera/thermo/SurfPhase.h"
+#include "cantera/thermo/ThermoFactory.h"
+#include "cantera/base/stringUtils.h"
+#include "cantera/base/utilities.h"
+#include "cantera/thermo/Species.h"
+
+using namespace std;
+
+namespace Cantera
+{
+
+CoverageDependentSurfPhase::CoverageDependentSurfPhase(const std::string& infile,
+                                                       const std::string& id_):
+    m_theta_ref(1.0),
+    m_stateNumlast(-2)
+{
+    setNDim(2);
+    initThermoFile(infile, id_);
+}
+
+void CoverageDependentSurfPhase::addInterpolativeDependency(const
+                                                            InterpolativeDependency&
+                                                            int_deps)
+{
+    double hcov_last = 0.0;
```

Comment:
In cases where the method name is so long, we also use the style:
```suggestion
void CoverageDependentSurfPhase::addInterpolativeDependency(
    const InterpolativeDependency& int_deps)
{
    double hcov_last = 0.0;
```
It's especially desirable not to split a single argument's type + name across lines.